src/components/NavaMenu.js

```diff
@@ -0,0 +1,26 @@
+import RoutingButton from './RoutingButton';
+import Login from './Login';
+//import CurrentElo from './CurrentElo';
+import { useContext } from 'react';
+import AppContext from './AppContext';
+import { BottomNavigation, BottomNavigationAction } from '@mui/material';
+import HomeIcon from '@mui/icons-material/Home';
+import TimelineIcon from '@mui/icons-material/Timeline';
+import RoutingAction from './RoutingAction';
+
+
+function NavaMenu(props) {
```

Comment:
What is a "Nava"? I'm familiar with "Nav" obvs. but have never heard of "Nava".

Reply:
Changed navaMenu to navMenu